options/locale/locale_en-US.ini

```
@@ -1585,6 +1586,13 @@ teams.add_nonexistent_repo = "The repository you're trying to add does not exist
 teams.add_duplicate_users = User is already a team member.
 teams.repos.none = No repositories could be accessed by this team.
 teams.members.none = No members on this team.
+teams.specific_repositories = Specific repositories
+teams.specific_repositories_helper = Members will only have access to repositories explicitly added to the team.
```

```suggestion
teams.specific_repositories_helper = Members will only have access to repositories explicitly added to the team. Selecting this <strong>will not</strong> automatically remove repositories already added with "All repositories".
```

What about this, then?

Because as a user, _I do_ expect selecting this to remove the repos from the team. 😄

---

Sure, good to be clear. I will update tomorrow, not at computer tonight.

---

I've updated the help texts.